app/controllers/application_controller.rb

```diff
@@ -1,2 +1,15 @@
 class ApplicationController < ActionController::API
+  def authenticate_with_token
```

Comment:
http://tutorials.pluralsight.com/ruby-ruby-on-rails/token-based-authentication-with-ruby-on-rails-5-api

You can read this piece on how to expand your usage of token based authentication in a rails application :)